spec.bs

```diff
@@ -21,6 +22,15 @@ spec: ECMASCRIPT; urlPrefix: https://tc39.es/ecma262/
 spec: URL; urlPrefix: https://url.spec.whatwg.org/
   type: dfn
     text: serialize an integer; url: #serialize-an-integer
+    text: url; url: #concept-url
```

Comment:
These should not be necessary. What happens if you remove them?

Reply:
I do not understand the reason but when I newly add steps that use [=url=] or [=url/protocol=] etc, bikeshed raised the error.  However, I know they are used in the match algorithm already.  I tried to avoid adding steps and merge the usage of them into the existing steps to avoid unexpected reference errors.

Reply:
I went with workaround, and I do not need these lines any more.